common/on_actions/imperia_on_half_yearly_pulse.txt

```diff
@@ -0,0 +1,22 @@
+﻿on_imperia_half_yearly_pulse_country = {
+	effect = {
+		# Resets the global list twice per year only
+		# Instead of [every_country -> every_country] x yearly
+		if = {
+			limit = {
+				has_global_highest_gdp = yes
```

Comment:
could also add labor_movement_researched

Reply:
True! I'll change it!